indico/modules/events/contributions/forms.py

```diff
@@ -183,6 +183,14 @@ def validate_duration(self, field):
                 raise ValidationError(error_msg)
 
 
+class ContributionDefaultDurationForm(IndicoForm):
+    duration = TimeDeltaField(_('Duration'), [DataRequired(), MaxDuration(timedelta(days=1))],
+                              units=('minutes', 'hours'))
+
+    def __init__(self, *args, **kwargs):
```

Comment:
this override isn't needed since it doesn't do anything besides calling the parent

Reply:
oops I removed it and added it back while I was testing something